fastestimator/op/tensorop/model/update.py

```diff
@@ -32,35 +34,70 @@ class UpdateOp(TensorOp):
 
     Args:
         model: Model instance compiled by fe.build.
-        loss_name: The name of loss.
+        loss_name: The input loss key.
+        gradients: The gradients key which the model will update according to. The provided gradients will be directly
```

Comment:
An optional key containing model gradients. These will be directly applied to the model weights during an update. If not provided, gradients will be computed based on the specified `loss_name`, which will automatically handle any desired mixed-precision scaling. This argument may not be used if mixed-precision training is enabled.

Reply:
I change the last sentence to "This argument 'should' not be used if mixed-precision training is enabled." since we will raise an error when mixed-precision and gradients argument are detected.

Reply:
fixed